packages/fxa-content-server/app/scripts/models/auth_brokers/oauth-webchannel-v1.js

```diff
@@ -103,6 +103,12 @@ const OAuthWebChannelBroker = OAuthRedirectAuthenticationBroker.extend({
    */
   sendOAuthResultToRelier(result, account) {
     if (this.hasCapability('supportsPairing') || result.action === 'pairing') {
+      if (!this.relier.has('service')) {
+        // the service in the query parameter currently overrides the status message
+        // this is due to backwards compatibility
+        this.relier.set('service', this.relier.get('clientId'));
+        this._metrics.setService(this.relier.get('clientId'));
+      }
```

Comment:
nit: these few lines are identical to the ones above.  It would be nice to have them in one place.

Reply:
Since I am not 100% this will fix the above issue, I'll probably wait to clean this up.